admin/local/docker/compose/docker-compose.yml

```diff
@@ -348,11 +354,13 @@ services:
           entrypoint --log-level DEBUG replication-controller
           --db-uri mysql://qsreplica@repl-mgr-db:3306/qservReplica
           --db-admin-uri mysql://root:CHANGEME@repl-mgr-db:3306/qservReplica
-          --qserv-czar-db mysql://root:CHANGEME@czar-db:3306/qservMeta
-          --instance-id qserv_proj
           --worker qserv_worker_db=mysql://qsmaster@worker-db-0:3306/qservw_worker,host=worker-repl-0
           --worker qserv_worker_db=mysql://qsmaster@worker-db-1:3306/qservw_worker,host=worker-repl-1
           --xrootd-manager manager-xrootd
+          --
+          --instance-id=qserv_proj
+          --qserv-czar-db=mysql://root:CHANGEME@czar-db:3306/qservMeta
+          --http-root=/usr/local/qserv/www
```

Comment:
In practice, this option (`--http-root`) is de-facto mandatory in all Qserv deployments. Hence it should be present in the command template and (normally, unless one is experimenting with some non-standard configuration) be not present in the Docker compose config because a location of the root folder is fixed within the `lite` container: `/usr/local/qserv/www`.

If you think a user should have an option to override its value then you could move this option "to the left", before `--`.

Reply:
All options at this level are known to & settable by the user, (IMO) the `--` does not factor into that. (As above, `--` separates options known to `entrypoint` and those passed unchanged to the binary).

If you think `--http-root` belongs in the command template we can do that, although at that point, would it make sense to just make "/usr/local/qserv/www" a default value for `--http-root` in `replication-controller`?

Reply:
Regarding setting some reasonable default value for the folder in the controller's implementation. That would result in starting the `qhttp` service. Right now the service would only start if the non-empty option `--http-root` is provided. Perhaps we should fix that.